client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -158,9 +157,9 @@ export const StockLineForm: FC<StockLineFormProps> = ({
         <StyledInputRow
           label={t('label.batch')}
           Input={
-            <BasicTextInput
-              defaultValue={draft.batch ?? ''}
-              onChange={e => debouncedUpdate({ batch: e.target.value })}
+            <BufferedTextInput
```

Comment:
I figure this is mostly likely to break Ruru work so best for them to confirm @andreievg @CarlosNZ @lache-melvin  (we could make the teams in github proper and be able to tag them 🤔) 

Otherwise looks legit!

Reply:
why would it break? I'm not seeing how it is any different, except that the input internal state is now correct.

anyway, that was a side comment. the important thing is making a GH team for the oms teams! genius 🙏

Reply:
Sorry poor phrasing on my part - if the merge conflict resolution _were_ to break something here, Ruru own the latest changes here I think so thought they'd best be suited to judge.

FWIW I assume the changes in the last commit contains all the conflicts https://github.com/msupply-foundation/open-msupply/pull/3816/commits/e24159d3c4a184d7d2b72519224bb6e8b15960f9

Reply:
@msupply-foundation/ruru as @Chris-Petty said, you may want to double check this - after merge is fine, it's only going into develop 🙏